src/entities/relay.ts

```diff
@@ -0,0 +1,70 @@
+import { type DocumentLoader, Follow, Undo } from "@fedify/fedify";
+
+// Use fixed username and id for the relay client actor
+// Why use fixed username and id?
+// Because the relay client actor is a special actor that is not supposed to be interacted with by users.
+// Also, if we delete the a relay and send the Undo activity, the relay client needs to be online.
+// For example, AodeRelay first fetches the actor from our server again, and if we already deleted the actor, it will fail
+// and the realy keeps our server in the database.
+// And because we won't receive an activity to check whether the Undo was successful, we can't delete the relay client actor safely.
+export const HOLLO_RELAY_ACTOR_ID = "8a683714-6fa2-4e53-9f05-b4acbcda4db7";
+export const HOLLO_RELAY_ACTOR_USERNAME = "hollo-relay-follower";
```

Comment:
I think it would be better if it's another username which is harder to be duplicate, for example, having some special characters?

Reply:
Like `$hollo~relay~follower$`? Should be valid https://www.w3.org/community/reports/socialcg/CG-FINAL-apwf-20240608/#names

Reply:
That would be a good name!